pkg/pillar/scripts/device-steps.sh

```diff
@@ -302,7 +302,9 @@ wait_for_touch nim
 touch "$WATCHDOG_FILE/nim.touch"
 
 # Print diag output forever on changes
-$BINDIR/diag -f -o /dev/console runAsService &
+mkfifo /run/diag.pipe
```

Comment:
Would make sense to add a comment saying that the output can be killed by killing this tail process.

Reply:
done